### ISSUE TITLE
load init_net.pb and predict_net.pb into same graph for caffe2

### ISSUE THREAD

**Issue by jnorwood** [lutzroeder/netron#209]
I'm trying to view the caffe2 quantized mobilenetv2 model from this link:
https://github.com/caffe2/models/tree/master/mobilenet_v2_quantized

Is there a way to load in the predict_net.pb and init_net.pb into the same viewer?

https://caffe2.ai/docs/zoo.html

> Loading up a pre-trained model to do things like predictions such as object detection is very simple in Caffe2. You need two files: 1) a protobuf that defines the network, and 2) a protobuf that has all of the network weights. The first is generally referred to as the predict_net and the second the init_net.

**Comment by jnorwood** [comment]
I can load the two files separately without error.  Both are in my downloads folder on Windows 10.  However, the init_net.pb doesn't appear to be loading when I open predict_net.pb, as all the weight and bias tensor values are empty.  I'm using the 2.7.7 build installed today.

**Comment by jnorwood** [comment]
oh, I see, so even some init_net.pb values, from that file loaded alone, are going to be incorrect, since your pull is fixing some new data fill cases.  

I haven't tried to use caffe2 protobuf before, but I see they also support int64 and double_data storage types.  Maybe those would need to be handled as well.

https://github.com/pytorch/pytorch/blob/master/caffe2/proto/caffe2.proto

**Comment by jnorwood** [comment]
ok, I see you already handle int64, also it looks like this should be opening the init_net.pb,  so maybe some error is occurring, and I'm just not seeing it. I'm attaching a screen capture.
<img width="576" alt="predict" src="https://user-images.githubusercontent.com/1499591/50427800-e5234300-0875-11e9-9e90-f900f30eeeb8.png">



```
            caffe2.Metadata.open(host, (err, metadata) => {
                context.request('init_net.pb', null, (err, data) => {
                    var init = null;
                    if (!err && data) {

```

**Comment by lutzroeder** [comment]
`2.7.7` doesn't include 6ee3620. Are you running from TIP?

```
git clone https://github.com/lutzroeder/netron.git
cd netron
npm start
``` 

<img width="1136" alt="screenshot" src="https://user-images.githubusercontent.com/438516/50427861-48e94400-0855-11e9-8743-a29d5d38c69c.png">

**Comment by jnorwood** [comment]
Yes, the changes fix the load of the filter and bias values.  If the Y_zero_point parameters are the quantization offsets, they seem believable.  If the Y_scale values are supposed to be the quantization scale values, then they probably are not correct, since values are 0.  Thanks for the quick fix.
<img width="584" alt="predict2" src="https://user-images.githubusercontent.com/1499591/50431189-79010900-088e-11e9-92a0-001eb12566c9.png">
